src/main/java/org/truffleruby/language/TruffleBootNodes.java

```diff
@@ -177,7 +176,7 @@ private Source loadMainSourceSettingDollarZero(VirtualFrame frame) {
                         break;
 
                     case PATH:
-                        final DynamicObject path = (DynamicObject) getFindSFileSnippetNode().execute(
+                        final DynamicObject path = (DynamicObject) findSFileSnippetNode.execute(
                                 frame,
                                 "Truffle::Boot.find_s_file Truffle::Boot.get_option('to_execute')");
```

Comment:
This is actually also just one call since the option could be trivially accessed from Java.